### DISCUSSION TITLE
overlap is not added on png image

### DISCUSSION THREAD

**Discussion by petdud** [libvips/libvips#3933] [Q&A]
I have an image in PNG format with some transparency - 25mb 70000x40000

When I use this library to transform it to JPG file with overlap 2, everything is amazing.

But my goal is to keep the transparency, so I use PNG suffix with same overlap, but with the PNG there are micro lines when displaying it with openseadragon. I assume there is something wrong here as JPG works fine.

`vips dzsave large_combined.png final --suffix .jpg --overlap 2` - works perfectly
`vips dzsave large_combined.png final --suffix .png --overlap 2` - has micro lines that shouldn't be there

Could you please help me how to fix it?

Here you can see the micro lines:
<img width="847" alt="Screenshot 2024-04-13 at 21 56 02" src="https://github.com/libvips/libvips/assets/13124780/29f9705b-6397-4b2b-ba67-5b58c9c1ddd7">

Version: 8.15.2

**Comment by jcupitt** [comment]
Hello @petdud,

I'd have a careful look at the tiles and verify that the positions and overlaps are correct.

If `dzsave` is creating the PNG tiles correctly (I think it probably is), then it must be an openseadragon bug and you need to open an issue there.

**Comment by petdud** [comment]
@jcupitt thank you for looking into it - you were right, it's on openseadragon. If anyone would have the same issue, here is a github link where is a solution: https://github.com/openseadragon/openseadragon/issues/2515 🙏
